llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -737,7 +737,7 @@ define void @foo(i1 %cond0, i1 %cond2) {
 TEST_F(SandboxIRTest, LoadInst) {
   parseIR(C, R"IR(
 define void @foo(ptr %arg0, ptr %arg1) {
-  %ld = load i8, ptr %arg0, align 64
+  %ld = load volatile i8, ptr %arg0, align 64
```

Comment:
Nit: Let's keep both the original non-volatile load and the volatile load, and `EXPECT_FALSE(Ld->isVolatile())` for the first one and `EXPECT_TRUE(VolLd->isVolatile())`.

Reply:
Ah okay, that makes sense. So the end result would be like this?

```
%ld = load i8, ptr %arg0, align 64
%vld = load volatile i8, ptr %arg0, align 64
```

Reply:
yes, this looks great.

Reply:
I fixed it